src/js/components/covid19/sankey/Sankey.jsx

```diff
@@ -0,0 +1,257 @@
+/**
+ * SankeyViz.jsx
+ * Created By Jonathan Hill 06/05/20
+ */
+
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { uniqueId } from 'lodash';
+import { TooltipWrapper } from 'data-transparency-ui';
+import { Glossary } from 'components/sharedComponents/icons/Icons';
+import { TooltipComponent } from 'containers/covid19/helpers/covid19';
+import { calculateUnits, formatMoneyWithPrecision } from 'helpers/moneyFormatter';
+import {
+    otherSankeyNodes,
+    dataForLinks,
+    defCodeColor
+} from 'dataMapping/covid19/covid19';
+import SankeyNode from './SankeyNode';
+import SankeyLink from './SankeyLink';
+
+const isIE = !!document.documentMode;
```

Comment:
this is pretty neat. we should make this a global constant

Reply:
Let us do that when we actually use it more than once.